dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -61,6 +58,10 @@ cookie.same.site.lax=Set the session cookie SameSite attribute value to Lax
 cookie.same.site.none=Set the session cookie SameSite attribute value to None
 cookie.same.site.strict=Set the session cookie SameSite attribute value to Strict
 
+# The word Partitioned should not be translated
+cookie.partitioned=Include a Partitioned attribute in session cookies
+cookie.partitioned.desc=Specifies that session cookies include the partitioned attribute. Only applies when samesite none is set. The partitioned attribute will only be used by browsers which support cookie partitioning.
```

Comment:
Suggested rewrite:
cookie.partitioned.desc=Specifies that session cookies include the Partitioned attribute. Only applies when the SameSite attribute is set to none. The Partitioned attribute is only used by browsers that support cookie partitioning.

Reply:
Correction:
cookie.partitioned.desc=Specifies that session cookies include the Partitioned attribute. The Partitioned attribute only applies when the SameSite attribute is set to none, and is only used by browsers that support cookie partitioning.

Reply:
Thanks -- I've made the update